pkg/services/authz/rbac/service.go

```diff
@@ -374,3 +488,16 @@ func (s *Service) buildFolderTree(ctx context.Context, ns claims.NamespaceInfo)
 
 	return folderMap, nil
 }
+
+func getChildren(folderMap map[string]FolderNode, folderUID string) []string {
+	folder, has := folderMap[folderUID]
+	if !has {
```

Comment:
Interesting case. Would that be that you have a permission on a folder we didn't find in the tree?

Reply:
Yes, exactly. It should never happen, but I added it in just in case.